mlflow/utils/autologging_utils/safety.py

```diff
@@ -618,6 +629,14 @@ def _wrap_patch(destination, name, patch, settings=None):
 
     patch = gorilla.Patch(destination, name, wrapped, settings=settings)
     gorilla.apply(patch)
+    return patch
+
+
+def _store_patch(autologging_integration, patch):
```

Comment:
Can we add a small docstring for this method?

Reply:
Done